### PR TITLE
Adding reference for standard load profiles

### LINKED ISSUE #67: Add field for well known standard load profiles
Currently, it is not possible to attach information on which standard load profile may be applied to a certain node. Add this possibility.

### PR DESCRIPTION
**Pull request by ckittl**
Resolves #67

### CODE REVIEW

File: src/main/java/edu/ie3/datamodel/models/BdewLoadProfile.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/*
+ * © 2020. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.datamodel.models;
+
+/**
+ * German standard load profiles, defined by the bdew (Bundesverband der Energie- und
```
Comment (johanneshiry) [inline src/main/java/edu/ie3/datamodel/models/BdewLoadProfile.java]:
```suggestion
 * German standard electricity load profiles, defined by the bdew (Bundesverband der Energie- und
```

File: src/main/java/edu/ie3/datamodel/models/StandardLoadProfile.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.datamodel.models;
+
+import edu.ie3.datamodel.exceptions.ParsingException;
+import java.util.Arrays;
+
+/**
+ * Giving reference to a known standard load profile to apply to a {@link
```
Comment (johanneshiry) [inline src/main/java/edu/ie3/datamodel/models/StandardLoadProfile.java]: :+1: for this very good documentation what this is used for

File: src/main/java/edu/ie3/datamodel/models/StandardLoadProfile.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+ * edu.ie3.datamodel.models.input.system.LoadInput}. This interface does nothing more, than giving a
+ * reference, the values have to be provided by the simulator using the models.
+ *
+ * <p>If you intend to provide distinct values, create either an {@link
+ * edu.ie3.datamodel.models.timeseries.IndividualTimeSeries} or {@link
+ * edu.ie3.datamodel.models.timeseries.RepetitiveTimeSeries} and assign it to the model via mapping
+ * to the model.
+ */
+public interface StandardLoadProfile {
```
Comment (johanneshiry) [inline src/main/java/edu/ie3/datamodel/models/StandardLoadProfile.java]: 👍 for the interface and the `DefaultLoadProfiles` implementation

File: src/main/java/edu/ie3/datamodel/models/input/system/LoadInput.java [hunk 4]
```diff
@@ -17,6 +18,13 @@
 
 /** Describes a load */
 public class LoadInput extends SystemParticipantInput {
+  /**
+   * Reference to a standard load profile to use for the model. If you intend to assign specific
+   * values, create an {@link edu.ie3.datamodel.models.timeseries.IndividualTimeSeries} or {@link
+   * edu.ie3.datamodel.models.timeseries.RepetitiveTimeSeries} and assign it via mapping to this
+   * model
+   */
```
Comment (johanneshiry) [inline src/main/java/edu/ie3/datamodel/models/input/system/LoadInput.java]:
```suggestion
  /**
   * Reference to a standard load profile to use for the model. If you intend to assign specific
   * values, create an {@link edu.ie3.datamodel.models.timeseries.IndividualTimeSeries} or {@link
   * edu.ie3.datamodel.models.timeseries.RepetitiveTimeSeries} and assign it via an external mapping (e.g. by providing a global time series for a specific load profile) to this model
   */
```

File: src/test/groovy/edu/ie3/datamodel/models/StandardLoadProfileTest.groovy [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    def "Standard load profiles are parsed correctly from correct input" () {
+        when:
+        StandardLoadProfile actual = StandardLoadProfile.parse(key)
+
+        then:
+        actual == expected
+
+        where:
+        key     || expected
+        "h0"    || BdewLoadProfile.H0
```
Comment (johanneshiry) [inline src/test/groovy/edu/ie3/datamodel/models/StandardLoadProfileTest.groovy]: 👍  for this test - documents all allowed cases for other developers/users and also tests them - nice work!

**Comment by ckittl** [comment]
!test

**Comment by ckittl** [comment]
!test

**Comment by johanneshiry** [comment]
!test

**Pull request merged**